src/Cli/dotnet/commands/dotnet-run/RunCommand.cs

```diff
@@ -240,9 +256,18 @@ private ICommand GetTargetCommand()
             var project = new ProjectInstance(Project, globalProperties, null);
 
             string runProgram = project.GetPropertyValue("RunCommand");
+            string targetFrameworks = project.GetPropertyValue("TargetFrameworks");
             if (string.IsNullOrEmpty(runProgram))
             {
-                ThrowUnableToRunError(project);
+                if (string.IsNullOrEmpty(Framework) && !string.IsNullOrEmpty(targetFrameworks))
+                {
+                    Framework = targetFrameworks.Split(';', StringSplitOptions.RemoveEmptyEntries | StringSplitOptions.TrimEntries).First();
```

Comment:
I'm wondering if we should have some kind of a validity check in here...I think it's unnecessary because you can't build something with TargetFrameworks="nonsense;net6", and the RemoveEmptyEntries part would handle TargetFrameworks=";net6"; is that correct?